src/Sulu/Bundle/MediaBundle/Media/ImageConverter/Command/ScaleCommand.php

```diff
@@ -21,7 +21,8 @@ public function execute(&$image, $parameters)
     {
         $size = $image->getSize();
 
-        $retina = isset($parameters['retina']) && $parameters['retina'] != 'false' ? 2 : 1;
+        $retina = isset($parameters['retina']) && $parameters['retina'] ? 2 : 1;
+        $forceRatio = !isset($parameters['forceRatio']) ? true : $parameters['forceRatio'];
```

Comment:
For me this reads better:

````php
$forceRatio = isset($parameters['forceRatio']) ? $parameters['forceRatio'] : true;
````

I would also consider:

````php
$parameters = array_merge(array(
    'retina' => 1,
    'forceRatio' => true,
    'x' => null,
    'y' => null,
), $parameters);
````

That way you clearly define the default values and do not have to check if things are set.